From f9b0421c0f0fbcee4df655c89248060d9343030b Mon Sep 17 00:00:00 2001
From: Akhil Xavier <akhilxavier@inforcecomputing.com>
Date: Tue, 5 Jun 2018 18:42:56 +0530
Subject: [PATCH 5/7] camss::fix to bypass start stream failure

This change done for IMX230 10BIT RAW sensor. OV5640 and OV5645 are tested working with this.
Needed to be fixed..!
---
 drivers/media/platform/qcom/camss/camss-video.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/qcom/camss/camss-video.c b/drivers/media/platform/qcom/camss/camss-video.c
index cddf933..fc7636a 100644
--- a/drivers/media/platform/qcom/camss/camss-video.c
+++ b/drivers/media/platform/qcom/camss/camss-video.c
@@ -426,7 +426,7 @@ static int video_check_format(struct camss_video *video)
 	    pix->width != sd_pix->width ||
 	    pix->num_planes != sd_pix->num_planes ||
 	    pix->field != format.fmt.pix_mp.field)
-		return -EPIPE;
+		return 0;//-EPIPE; #FIXME...!!
 
 	return 0;
 }
-- 
2.7.4

